include/nuraft_mesg/mesg_state_mgr.hpp

```diff
@@ -28,6 +28,15 @@ struct replica_config {
     std::string aux;
 };
 
+struct peer_info {
+    // Peer ID.
+    std::string id_;
+    // The last log index that the peer has, from this server's point of view.
+    ulong last_log_idx_;
```

Comment:
I've tried to avoid using non-fixed length fields...I'm not really sure when they would be ideal when we know the max size of log idx...it's a u64 yeah?

Reply:
yes , will change it to u64